### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by leesunmyung** [CVPR2023-3D-Occupancy-Prediction/CVPR2023-3D-Occupancy-Prediction#61]
Thank you for your interesting work.

I want to ask you about loss function.

Following the paper, you said that OHEM loss was used for model training.

However, I can't find where it is implemented in the code.

In 'bevformer_base_occ.py', just cross-entropy loss was used.

Could you let me know where the OHEM loss is implemented?

**Comment by focus-on-11-road** [comment]
您的来信已收到，非常感谢！我将尽快对您的来信给予回复，祝您工作生活愉快！
                                                  刘宇杰
